my-express-app/models/video.js

```diff
@@ -17,4 +17,12 @@ export const Videos = sequelize.define("Videos", {
     type: DataTypes.TEXT,
     allowNull: false,
   },
+  AgeGroup: {
+    type: DataTypes.ENUM("5-7", "7-9", "9-11"),
+    allowNull: false,
+  },
+  Interest: {
+    type: DataTypes.STRING,
```

Comment:
I see now why you were getting issues with matching interests in the algorithm. Seeing how the interest field was populated was inconsistent (using ands, using commas, using spaces to separate each interest). A better way to have seeded the data base was to store Interests as a list of strings! 

You were able to get around this in your actual algorithm, but if you have any time (after you finish the project requirements or maybe in week 9) I would see if you could reseed the database, with interests looking like:

From `"Science and Technology Projects for Kids , Robotics and Mechanical Engineering "` to `["Science and Technology Projects for Kids", "Robotics and Mechanical Engineering"]`


From `"Science and Technology Projects for Kids"` to `["Science and Technology Projects for Kids"]`